internal/service/encryptionatrest/resource_migration_test.go

```diff
@@ -131,19 +141,16 @@ func TestMigEncryptionAtRest_basicAzure(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				ExternalProviders: mig.ExternalProviders(),
-				Config:            acc.ConfigEARAzureKeyVault(projectID, &azureKeyVault, false),
+				Config:            acc.ConfigEARAzureKeyVault(projectID, &azureKeyVault, false, false), // not using data source as it was introduced in 1.19.0
```

Comment:
chose to not use data source for migration tests instead of skipping the tests for lower versions

Reply:
you can use mig.IsProviderVersionAtLeast instead of directly false, you can look for some examples in the code